server/validators/goalTargetCompletionDateValidator.ts

```diff
@@ -29,3 +35,7 @@ export default function goalTargetCompletionDateValidator(
 
   return errors
 }
+
+const isNotNumeric = (value: string): boolean => {
```

Comment:
This is how we catch some of the weird cases (such as `][` apparently being a valid day component !!!!)